Gemfile

```diff
@@ -26,6 +26,7 @@ gem "sass-rails", "~> 5.0"
 gem "turbolinks", "~> 5"
 gem "uglifier", ">= 1.3.0"
 gem "will_paginate", "~> 3.1.7"
+gem "pundit"
```

Comment:
em dùng pundit để làm gì z em

Reply:
pundit để phân quyền hoặc là để scope một model theo từng role khác nhau anh...

Reply:
ví dụ User model có hàm destroy thì chỉ có admin mới có quyền này...
ví dụ khi lấy tất cả user lên view thì ...admin lấy all user kể cả user chưa actice...còn role Trainer chỉ lấy đc user đã activated thông qua Scope.

Reply:
trong project 1 chỉ dùng các gem trong tutorial đã dùng, còn lại code chay nhé chú, ko chơi gem nà

Reply:
em có chơi namespace :D ! pundit em chưa dùng nhiều chị ạ! :D pull này toàn dùng namespace thôi :D...

Reply:
vậy là xóa hay để vâyj đến project 2 luôn đây chị!

Reply:
xóa gem đi, dùng namespace thôi nhé :p
lần sau nhưng xác nhận thế này, to thẳng lên box để dc phản hồi nhanh nhất nhé